apps/src/p5lab/spritelab/commands/spriteCommands.js

```diff
@@ -70,7 +66,11 @@ export const commands = {
   },
 
   makeNumSprites(num, animation) {
-    num = Math.min(num, BIG_NUMBER_GUARD);
+    var numSprites = this.getNumberOfSprites();
+    if (numSprites >= MAX_NUM_SPRITES) {
```

Comment:
This is another place where we could use a helper function.

Reply:
Thanks - made the update!